Sources/PointFree/Home.swift

```diff
@@ -11,25 +11,23 @@ import UrlFormEncoding
 
 let secretHomeMiddleware: (Conn<StatusLineOpen, Database.User?>) -> IO<Conn<ResponseEnded, Data>> =
   writeStatus(.ok)
-    >-> respond(secretHomeView.map(addGoogleAnalytics))
+    >-> respond(
+      view: secretHomeView,
+      layoutData: { currentUser in
+        SimplePageLayoutData(
+          currentUser: currentUser,
+          data: currentUser,
+          extraStyles: pricingExtraStyles,
+          showTopNav: false,
+          title: "Point-Free: A weekly video series on functional programming and the Swift programming language."
```

Comment:
Realizing there's a bunch of `meta` data we probably want to be able to customize here!

Reply:
yup yup, we have a metadata layout we can compose with this and `contramap` on, or we could just add all those fields to this struct